dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/icon/IconController.java

```diff
@@ -130,15 +132,24 @@ public void getIconData(@PathVariable String key, HttpServletResponse response)
   }
 
   @GetMapping
-  public @ResponseBody List<ObjectNode> getAllIcons(IconRequestParams iconRequestParams)
+  public @ResponseBody PaginatedIconResponse getAllIcons(IconRequestParams iconRequestParams)
       throws BadRequestException {
 
     IconOperationParams iconOperationParams = iconRequestParamsMapper.map(iconRequestParams);
 
+    Pager pager = iconOperationParams.getPager();
```

Comment:
Why are you creating this var here? Always declare it as close to your usage. For this particular case I think you can even inline it as it does not add any value.